### PR TITLE
requested changes for clarity

### PR DESCRIPTION
**Pull request by iSpeakNerd**

### CODE REVIEW

**Comment by iSpeakNerd** [comment]
review please @uniyalabhishek

File: help-docs/FAQ/join-tg-group.md [hunk 1]
```diff
@@ -36,38 +36,46 @@ For best results:
...
+3. Click the `START` button
+
+4. Verify the `Community:` field is correct for the community you are trying to join.
 
 <div class="text--center">
   <img src={img3a} alt="Verify Community details" width="400" />
 </div>
 
-4. Click the "Connect Wallet" button from the bot.
+5. Click the "Connect Wallet" button from the bot.
```
Comment (uniyalabhishek) [inline help-docs/FAQ/join-tg-group.md:52]: should we mention that you can "use existing wallets" or "add a new wallet" - and then "Connect Wallet"?
image

File: help-docs/FAQ/join-tg-group.md [hunk 2]
```diff
@@ -36,38 +36,46 @@ For best results:
...
+:::
+
+6. In your browser popup, select a wallet from the list of [Collab.Land-support wallets](/help-docs/wallets/verify-your-wallet#supported-wallets) to verify.
 
 <div class="text--center">
   <img src={img4} alt="Select your wallet to verify" width="300"/>
 </div>
 
-6. Follow the instructions to have your wallet sign the connection request from the bot.
+7. Follow the instructions to have your wallet sign the connection request from the bot.
```
Comment (uniyalabhishek) [inline help-docs/FAQ/join-tg-group.md:70]: regarding this, we have some good language here too https://docs.collab.land/help-docs/wallets/verify-your-wallet, except this line "This step is necessary to confirm your identity and ensure the security of your funds. After the first time, you will have the option to choose an existing connected wallet without having to sign a new message."

**Review by uniyalabhishek — CHANGES_REQUESTED** [review]
added a couple of comments

File: help-docs/FAQ/join-tg-group.md (hunk 2 above)
Comment (uniyalabhishek) [reply help-docs/FAQ/join-tg-group.md:70]: something we mention in support:

Collab.Land is a read-only application. When you sign a message, you attest that you own that particular wallet address. Collab.Land reads the public blockchain to verify that the wallet address is connected to the required token address to grant a role. There is no opportunity for Collab.Land to have access to anything other than your public wallet address, which is viewable to everyone.

Comment (iSpeakNerd) [reply help-docs/FAQ/join-tg-group.md:77]: > something we mention in support:
> 
> Collab.Land is a read-only application. When you sign a message, you attest that you own that particular wallet address. Collab.Land reads the public blockchain to verify that the wallet address is connected to the required token address to grant a role. There is no opportunity for Collab.Land to have access to anything other than your public wallet address, which is viewable to everyone.

except we're adding write capabilities with gmPASS

Comment (iSpeakNerd) [reply help-docs/FAQ/join-tg-group.md:77]: added it but we'll have to change that very soon

**Review by iSpeakNerd — COMMENTED** [review]
i don't know what this comment is for

**Review by uniyalabhishek — APPROVED** [review]
thank you! @iSpeakNerd

**Pull request merged**